version/version.go

```diff
@@ -5,7 +5,7 @@ var (
 	Package = "github.com/containerd/containerd"
 
 	// Version holds the complete version number. Filled in at linking time.
-	Version = "1.0.1-rc.0+unknown"
+	Version = "1.0.1+unknown"
```

Comment:
is this supposed to keep the `-unknown`?  Probably a stupid question but just thought I would double check.

Reply:
it gets overwritten here: https://github.com/containerd/containerd/blob/master/Makefile#L44

Reply:
The goal of keeping this with `+unknown` is so that builds done without the linker injection are easily identified.